src/templates/post/index.jsx

```diff
@@ -36,19 +39,19 @@ const PostTemplate = ({ pageContext, data }) => {
             cover={post.cover}
             date={data.markdownRemark.fields.date}
             html={postNode.html}
+            category={post.category}
           />
         </MainPost>
-        <News>
-          <NewsCard items={news} />
-        </News>
+        {panel}
         {otherPosts.map((post) => (
-          <PostContainer>
+          <PostContainer key={post.slug}>
             <PostCard
               description={post.description}
               slug={post.slug}
               title={post.title}
               date={post.date}
               author={post.author}
+              category={post.category}
```

Comment:
Tady tu `prop` bych odstranil a karty pod článkem bych nechal pouze pro články, protože tiskovky už máme na boční straně. Tedy bych přidal _blog_ filtr do query.

Reply:
Přidal jsem filtr, ale kategorii bych zatím nechal. Co když se rozhodneme mít více kategorií? Pak se filtr upraví na excluded press, a u ostatních článků se bude moct zobrazovat kategorie.

Reply:
OK, to dává smysl. 👍